deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -12,6 +12,11 @@
   notify:
     - Restart Nginx
 
+- name: Download Jenkins CLI jar
+  get_url:
```

Comment:
Addresed in [0132b8e](https://github.com/azavea/raster-foundry/pull/522/commits/0132b8e387b7f21e683409d88064739e4f212b39)